### DISCUSSION TITLE
bslib's dashboarding experience

### DISCUSSION THREAD

**Discussion by cpsievert** [rstudio/bslib#618] [Announcements]
With the 0.5.0 release of `{bslib}` (coming soon), we now view it as a viable alternative to `{shinydashboard}` for creating dashboards. For an overview of what's possible, see [this article](https://rstudio.github.io/bslib/articles/dashboards.html)

That said, we'd love to hear about what things you find are missing, especially if it's something that keeping you from adopting `{bslib}` over `{shinydashboard}`/`{bs4Dash}`

**Comment by teofiln** [comment]
Hi guys! Is it possible to create sidebar with navigation with the current release of `bslib v0.5.0`? I see a lot of discussion about this in the issues, but I am not sure if this is currently possible in some way and you are improving the API, or not possible at all yet?

I've tried some things but without luck.

```
page_sidebar(
  title = "Example dashboard",
  sidebar = sidebar(
    nav_panel("a", "A"),
    nav_panel("b", "B")
  )
)

page_fillable(
  title = "Example dashboard",
  layout_sidebar(
    nav_panel("a", "A"),
    nav_panel("b", "B")
  )
)
```

Thanks for the awesome package!

**Reply by gadenbuie** [reply]
This isn't possible... yet! We're planning on working on it soon and have an open issue to track progress on that front: https://github.com/rstudio/bslib/issues/585

The API we have planned will be similar to `layout_sidebar()`, but we intend to have a bit of separation between sidebars for navigation and sidebars with inputs and arbitrary content.

**Reply by teofiln** [reply]
Thanks @gadenbuie! Looking forward to it!

**Reply by MattGrnt** [reply]
Definitely something I'd be interested in, and would make our team more likely to move over to bslib from shinydashboard!

**Reply by Mosk915** [reply]
I would also be interested in sidebar navigation.  This is the biggest thing preventing us from switching over from shinydashboard.
